experimental/traffic-portal/src/app/api/cdn.service.ts

```diff
@@ -119,4 +119,16 @@ export class CDNService extends APIService {
 
 		return this.put<ResponseCDN>(`cdns/${id}`, body).toPromise();
 	}
+
+	/**
+	 * Queues or dequeues updates on a CDN's servers.
+	 *
+	 * @param cdn The CDN to queue or dequeue updates on.
+	 * @param action The action to perform on the CDN, either "queue" or "dequeue".
+	 */
+	public async queueCDNUpdates(cdn: ResponseCDN, action: "queue" | "dequeue"): Promise<CDNQueueResponse> {
+		const path = `cdns/${cdn.id}/queue_update`;
+		const request: CDNQueueRequest = {action};
+		return this.post<CDNQueueResponse>(path, request).toPromise();
+	}
```

Comment:
This method is not covered by the CDN API service's unit tests, which brings the service's coverage down from 100% to 86.95% of statements.

Reply:
Added a `queueCDNUpdates()` test in 979020a30b